### DISCUSSION TITLE
suggestion: Link to a openstreetmap edit

### DISCUSSION THREAD

**Discussion by backache** [openbenches/openbenches.org#319] [Ideas]
To encourage linking between the two sites, how about on a benches pages (for example https://openbenches.org/bench/13510/) as well as the 

"benches near this" that links to https://openbenches.org/#51.365509/1.038294/16 

you have an 

"edit on openstreetmap" that links to https://www.openstreetmap.org/edit#map=22/51.365509/1.038294

This would be to encourage the information to be shared, but also perhaps for people  to put openbenches URL in the openstreetmaps website tag for the bench (as well as filling in the inscription tag etc)

**Comment by edent** [comment]
I love that idea - can you please submit a pull request for it?

**Comment by backache** [comment]
I'm not a coder nor do have I made a dev environment but I'll rough out the button, no guarantees it'll work though!

**Comment by edent** [comment]
I think it might be easier to link to MapComplete - for example https://mapcomplete.osm.be/benches.html?z=17&lat=51.49882&lon=0.127512

**Comment by backache** [comment]
That's wierd, when I see big purple numbers in each of those squares and when I click on them get a picture (it is their aim to have  a picture for every grid square the UK)
